warehouse/ingest-core/src/test/java/datawave/ingest/mapreduce/TableConfigHelperFactoryTest.java

```diff
@@ -55,6 +73,9 @@ private void recreateTable(TableOperations tops, String table) throws Exception
     
     @Test
     public void shouldSetupTableWithOverrides() throws Exception {
+        recreateTable(tops, TableName.SHARD);
```

Comment:
The `@Before` method would still get run, before every test, so I think it would be better to have these two lines in a method annotated with that, rather than in each test.

Reply:
Fixed in 3f4c0fe